examples/Workflows/densenet_image_classifier_pipeline/densenet_model/densenet_handler.py

```diff
@@ -0,0 +1,147 @@
+"""
```

Comment:
@dhanainme We need to document/add user guide for the differences of the handler for models(nodes) in the workflow with regular handlers we have in the non-workflow use-cases.